### ISSUE TITLE
Move from Discord to Slack

### ISSUE THREAD

**Issue by Zethson** [mlf-core/mlf-core#254]
Discord is just as bad as Slack privacy wise.

Nevertheless, Slack seems to be the standard in scientific chat communication and to blend in we will also resort to using Slack.

:pirate_flag:

**Comment by Zethson** [comment]
Apparently slack has a new "feature" (quote from their customer service to me lol) that you cannot create non expiring invitation links anymore.
I absolutely don't want to change the link every 30 days. That's just dumb.

What to do?

**Comment by Zethson** [comment]
I can now create unlimited invitation links again. In 11 days Slack premium expires and I know whether this is persistent or not.

**Comment by Emiller88** [comment]
We could be trendy and use matrix. Start a movement in the scientific community.

**Comment by Zethson** [comment]
> 
> 
> We could be trendy and use matrix. Start a movement in the scientific community.

@Imipenem and me are actually communicating over Matrix. Do you think that people would really join the Matrix network?
Also, can we easily create a non-expiring invite link to a Matrix room? Maybe https://matrix.to/ could work?
I've never done this before.

**Comment by Emiller88** [comment]
I think we may have to host it but the non-expiring link looks possible and the explanation is pretty quick. I haven't had a good excuse to use matrix yet myself. It looks like they use matrix.to here https://linuxunplugged.com/matrixinfo

**Comment by Zethson** [comment]
> 
> 
> I think we may have to host it but the non-expiring link looks possible and the explanation is pretty quick. I haven't had a good excuse to use matrix yet myself. It looks like they use matrix.to here https://linuxunplugged.com/matrixinfo

I def don't want to host a matrix server. However, Element allows you to create rooms, so we might be able to make use of Element's infrastructure?

Still unsure whether this generally is a good idea...

**Comment by Emiller88** [comment]
> I def don't want to host a matrix server
Just to clarify, is it that you don't want to admin it, or just don't want to bother with it as an organization?

Here's there's hosted platform pricing https://element.io/pricing which looks cheaper than slack.

**Comment by Zethson** [comment]
> 
> 
> > I def don't want to host a matrix server
> > Just to clarify, is it that you don't want to admin it, or just don't want to bother with it as an organization?
> 
> Here's there's hosted platform pricing https://element.io/pricing which looks cheaper than slack.

Yeah, but we don't have any funding to pay for this. Money is not an issue for my current groups, but I would have to ask for dedicated funding for this. Since most of the work was conducted while I was associated with QBiC I don't know whether I should pursue this...

Element itself is free and according to their pricing the free tier has:

```
Communities/groups of users

Unlimited
```

**Comment by Emiller88** [comment]
I'll have to play around with it. Does that include a "server" where we can have channels for the templates and whatnot like the discord is set up currently? Not just a single chat for a group.

**Comment by Zethson** [comment]
Ah yeah, good point. From what I can see it's only a single room and no sub channels/rooms.

**Comment by Zethson** [comment]
@Emiller88 just had a quick discussion with @Imipenem. Although we, as avid users, love the Matrix network we don't think that it's really suitable for our purpose at the moment.

It will be one of Discord, Slack, Mattermost I guess.

**Comment by Emiller88** [comment]
I agree sadly. Thanks for the update!
